src/linker/Linker/LinkContext.cs

```diff
@@ -585,6 +605,12 @@ public bool IsWarningSuppressed (int warningCode, MessageOrigin origin)
 
 			return Suppressions.IsSuppressed (warningCode, origin, out _);
 		}
+
+		public static WarnVersion GetWarningVersion (int code)
+		{
+			// TODO: Once we add warnings beyond .NET5, give them a new WarnVersion
```

Comment:
Nit: this is no TODO - at least to me :wink:

Reply:
Removed TODO and rephrased :)